ui/ui-components/components/model/overview/ModelOverviewDestinations.tsx

```diff
@@ -0,0 +1,74 @@
+import { ListGroup, ListGroupItem } from "react-bootstrap";
+import { Models } from "../../../utils/apiData";
+import { grouparooUiEdition } from "../../../utils/uiEdition";
+import DestinationCollectionLink from "../../destination/DestinationCollectionLink";
+import EntityInfoContainer from "./EntityInfoContainer";
+import EntityInfoHeader from "./EntityInfoHeader";
+import Link from "next/link";
+import EnterpriseLink from "../../../components/GrouparooLink";
+import { useGrouparooModelContext } from "../../../contexts/grouparooModel";
+import LinkButton from "../../LinkButton";
+import ModelOverviewCard from "./ModelOverviewCard";
+
+const DestinationInfo: React.FC<{ destination: Models.DestinationType }> = ({
+  destination,
+}) => {
+  const { app, connection, exportTotals } = destination;
+  const isCommunityUI = grouparooUiEdition() === "community";
```

Comment:
Could be hoisted outside of the component.

Reply:
I'm not sure what this function does, wether it's resolved at runtime or not